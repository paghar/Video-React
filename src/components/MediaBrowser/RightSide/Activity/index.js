//@flow
import React, { useState, useRef, useEffect } from 'react';
import ActivityItem from './ActivityItem';
import type { ActivityType } from './ActivityItem';
import TimeSeprator from './TimeSeprator';
import { getActivityTime, getToday } from 'helpers/time';
import { Scrollbars } from 'react-custom-scrollbars';
import { Generals } from 'theme/colors';
import { Shadow, StyledIcon, StyledThumb, StyledTrack } from './index.style';
type Props = {
  activities: Array<ActivityType>,
  className?: string,
};

const Activity = ({ activities, className }: Props) => {
  //Ref for ScrollBar
  const scrollBarRef = useRef();
  //State for showing shadow in scrollbar
  const [isShadowShown, setIsShadowShown] = useState(false);
  //detect if there is scrolling
  useEffect(() => {
    const currentScrollBar = scrollBarRef.current;
    if (currentScrollBar) {
      const scrollHeight = currentScrollBar.getScrollHeight();
      const clientHeight = currentScrollBar.getClientHeight();
      if (scrollHeight > clientHeight) {
        setIsShadowShown(true);
      }
    }
  }, [activities, setIsShadowShown]);
  //Hide the shadow box when scrolling
  const handleScrollStart = () => {
    setIsShadowShown(false);
  };
  //style of scroll thumb
  const renderThumb = ({ style }) => {
    return <StyledThumb style={style} />;
  };
  //style of scroll track
  const renderTrackVertical = ({ style }) => {
    return <StyledTrack style={style} />;
  };
  const activityList = activities.map((activity, id) => {
    const {
      name,
      createdAt,
      avatarSrc,
      text,
      avatarColor,
      shortName,
    } = activity;
    let prev = id ? getActivityTime(activities[id - 1]) : getToday();
    let current = getActivityTime(activities[id]);
    return (
      <React.Fragment key={id}>
        {(current !== prev || (id === 0 && current === prev)) && (
          <TimeSeprator time={createdAt} />
        )}
        <ActivityItem
          shortName={shortName}
          avatarSrc={avatarSrc}
          name={name}
          createdAt={createdAt}
          text={text}
          avatarColor={avatarColor}
        />
      </React.Fragment>
    );
  });
  return (
    <Scrollbars
      className={className}
      autoHide
      ref={ref => (scrollBarRef.current = ref)}
      renderThumbVertical={renderThumb}
      renderTrackVertical={renderTrackVertical}
      onScrollStart={handleScrollStart}
    >
      {activityList}
      {isShadowShown && (
        <Shadow>
          <StyledIcon fontSize="15px" color={Generals.vii} iconNumber="61560" />
        </Shadow>
      )}
    </Scrollbars>
  );
};

export default Activity;
