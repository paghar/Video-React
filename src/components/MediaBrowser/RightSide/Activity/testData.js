import image1 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
import image2 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
import image3 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
const testData = [
  {
    avatarSrc: '',
    shortName: 'SP',
    name: '1Sara Parker',
    createdAt: 1571295775000,
    text: 'Uploaded "aportman family"',
    avatarColor: 'gray',
  },
  {
    avatarSrc: image2,
    name: '2Amili Hilton',
    shortName: 'AH',
    createdAt: 1571209375000,
    text: 'Removed 3 files',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image3,
    name: '3Andrew Sunny',
    shortName: 'AS',
    createdAt: 1571209375000,
    text: 'Shared "aportman family" with Peter Brown.',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '4Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571027260000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '5Pablo Neroda',
    createdAt: 1571016460000,
    text: 'Commented on "winter fall"',
    shortName: 'PN',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '6Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571009260000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '7Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571002060000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '8Pablo Neroda',
    shortName: 'PN',
    createdAt: 1570963070000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '8Pablo Neroda',
    ShortName: 'PN',
    createdAt: 1570955870000,
    text: 'Commented on "winter fall"',
    color: '#fff',
  },
  {
    avatarSrc: image1,
    name: '9Pablo Neroda',
    ShortName: 'PN',
    createdAt: 1570836460000,
    text: 'Commented on "winter fall"',
    color: '#fff',
  },
];
export default testData;
