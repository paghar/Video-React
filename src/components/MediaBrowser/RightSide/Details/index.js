//@flow
import React from 'react';
import Typography from 'components/shared/Typography';
import { Display4 } from 'theme/fonts';
import { Generals } from 'theme/colors';
import { DetailWrapper } from './index.style';
const Details = () => {
  return (
    <DetailWrapper>
      <Typography font={Display4} hoverable color={Generals.vi}>
        Details
      </Typography>
    </DetailWrapper>
  );
};

export default Details;
