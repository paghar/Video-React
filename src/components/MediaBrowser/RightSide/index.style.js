//@flow
import styled from 'styled-components';
import { Tabs, TabList, Tab, TabPanel } from 'react-tabs';
import { Generals, Primary } from 'theme/colors';
export const TabsWrapper = styled(Tabs)`
  background-color: ${Generals.iii};
  display: flex;
  flex-direction: column;
  height: 100vh;
  padding-right: 5px;
`;

export const StyledTabList = styled(TabList)`
  list-style-type: none;
  display: flex;
  flex-direction: row;
  padding: 0px;
  margin: 0;
  align-content: center;
`;
export const TabName = styled(Tab)`
  align-content: center;
  text-align: center;
  align-items: center;
  justify-content: center;
  flex: 1;
  border-bottom: 2px solid ${Generals.v};
  padding: 22px 0 18px 0;
  user-select: none;
  cursor: pointer;
  line-height: 2px;
  &.is-selected {
    border-bottom: 3px solid ${Primary.Dim};
    color: ${Generals.viii} !important;

    span {
      color: ${Generals.viii} !important;
    }
  }

  &:focus {
    outline: none;
    border-bottom: 3px solid #005dff;
    color: white !important;
    line-height: 3px;
    span {
      color: white !important;
    }
  }
`;
export const ContainerTabPanel = styled(TabPanel)`
  padding-top: 20px;
  display: none;
  overflow-y: auto;
  overflow-x: hidden;
  flex: 1;
  &.is-selected {
    -webkit-flex: 1 1 auto;
    display: block;
  }
`;
