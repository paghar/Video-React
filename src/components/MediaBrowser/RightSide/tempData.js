import image1 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
import image2 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
import image3 from 'assets/image/christopher-campbell-rDEOVtE7vOs-unsplash.png';
import { Avatars } from 'theme/colors';
const temp = [
  {
    avatarSrc: '',
    shortName: 'SP',
    name: '1Sara Parker',
    createdAt: 1571295775000,
    text: 'Uploaded "aportman family"',
    avatarColor: Avatars[0],
  },
  {
    avatarSrc: image2,
    name: '2Amili Hilton',
    shortName: 'AH',
    createdAt: 1571209375000,
    text: 'Removed 3 files',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image3,
    name: '3Andrew Sunny',
    shortName: 'AS',
    createdAt: 1571209375000,
    text: 'Shared "aportman family" with Peter Brown.',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '4Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571027260000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '5Pablo Neroda',
    createdAt: 1571016460000,
    text: 'Commented on "winter fall"',
    shortName: 'PN',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '6Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571009260000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '7Pablo Neroda',
    shortName: 'PN',
    createdAt: 1571002060000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '8Pablo Neroda',
    shortName: 'PN',
    createdAt: 1570963070000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '8Pablo Neroda',
    shortName: 'PN',
    createdAt: 1570955870000,
    text: 'Commented on "winter fall"',
    avatarColor: '#fff',
  },
  {
    avatarSrc: image1,
    name: '9Pablo Neroda',
    shortName: 'PN',
    createdAt: 1570836460000,
    text: 'Commented on "winter fall"',
    avatarColor: '#eee',
  },
];
export default temp;
//TODO:I should remove this file
