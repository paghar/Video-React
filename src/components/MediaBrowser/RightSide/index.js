//@flow
import React from 'react';
import Activity from './Activity';
import type { ActivityType } from './Activity/ActivityItem';
import Details from './Details';
import Typography from 'components/shared/Typography';
import { Display1 } from 'theme/fonts';
import { Generals } from 'theme/colors';
import {
  TabsWrapper,
  StyledTabList,
  TabName,
  ContainerTabPanel,
} from './index.style';
type Props = {
  className?: string,
  activities: Array<ActivityType>,
};

const RightSide = ({ activities, className }: Props) => {
  return (
    <TabsWrapper
      selectedTabClassName="is-selected"
      selectedTabPanelClassName="is-selected"
      className={className}
    >
      <StyledTabList>
        <TabName>
          <Typography font={Display1} color={Generals.vii}>
            Activity
          </Typography>
        </TabName>
        <TabName>
          <Typography font={Display1} color={Generals.vi}>
            Details
          </Typography>
        </TabName>
      </StyledTabList>
      <ContainerTabPanel>
        <Activity activities={activities} />
      </ContainerTabPanel>
      <ContainerTabPanel>
        <Details />
      </ContainerTabPanel>
    </TabsWrapper>
  );
};

export default RightSide;
