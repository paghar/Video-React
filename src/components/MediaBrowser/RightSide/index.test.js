//@flow
import React from 'react';
import RightSide from '.';
import renderer from 'react-test-renderer';
import testData from 'components/MediaBrowser/RightSide/Activity/testData';
describe('<TimeSeprator />', () => {
  it('checks the UI', () => {
    const wrapper = renderer
      .create(<RightSide activities={testData} />)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
