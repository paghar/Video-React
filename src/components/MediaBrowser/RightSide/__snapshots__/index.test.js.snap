// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeSeprator /> checks the UI 1`] = `
.c3 {
  pointer-events: auto;
}

.c5 {
  pointer-events: none;
  -moz-user-select: none;
  -khtml-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c7 {
  outline: none;
  border: none;
  background: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 0px 27px 0px 20px;
  margin: 30px 0px;
}

.c9 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding-left: 7px;
  position: relative;
}

.c10 {
  outline: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  pointer-events: none;
}

.c12 {
  position: absolute;
  top: 0px;
  right: 0px;
}

.c11 {
  outline: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  pointer-events: none;
}

.c8 {
  background-color: gray;
  background-size: 100% 100%;
  width: 24px;
  height: 24px;
  border-radius: 100%;
  border: 0px solid #0d0d1a;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  outline: none;
}

.c13 {
  background-color: #fff;
  background-image: url(christopher-campbell-rDEOVtE7vOs-unsplash.png);
  background-size: 100% 100%;
  width: 24px;
  height: 24px;
  border-radius: 100%;
  border: 0px solid #0d0d1a;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  outline: none;
}

.c14 {
  background-color: background-image:url(christopher-campbell-rDEOVtE7vOs-unsplash.png);
  background-size: 100% 100%;
  width: 24px;
  height: 24px;
  border-radius: 100%;
  border: 0px solid #0d0d1a;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  outline: none;
}

.c6 {
  background: none;
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  text-align: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
}

.c16 {
  border-radius: 10px;
  width: 3px !important;
  opacity: 0.3;
}

.c15 {
  height: 100%;
  right: 0px;
  margin-top: 5px;
  border-radius: 10px;
  width: 2px !important;
  padding: 5px 0px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  padding-right: 5px;
}

.c1 {
  list-style-type: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px;
  margin: 0;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
}

.c2 {
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  text-align: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  border-bottom: 2px solid;
  padding: 22px 0 18px 0;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
  line-height: 2px;
}

.c2.is-selected {
  border-bottom: 3px solid;
  color: !important;
}

.c2.is-selected span {
  color: !important;
}

.c2:focus {
  outline: none;
  border-bottom: 3px solid #005dff;
  color: white !important;
  line-height: 3px;
}

.c2:focus span {
  color: white !important;
}

.c4 {
  padding-top: 20px;
  display: none;
  overflow-y: auto;
  overflow-x: hidden;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c4.is-selected {
  -webkit-flex: 1 1 auto;
  display: block;
}

<div
  className="c0"
  data-tabs={true}
  onClick={[Function]}
  onKeyDown={[Function]}
>
  <ul
    className="c1"
    role="tablist"
  >
    <li
      aria-controls="react-tabs-1"
      aria-disabled="false"
      aria-selected="true"
      className="c2 is-selected"
      id="react-tabs-0"
      role="tab"
      tabIndex="0"
    >
      <span
        className="c3"
      >
        Activity
      </span>
    </li>
    <li
      aria-controls="react-tabs-3"
      aria-disabled="false"
      aria-selected="false"
      className="c2"
      id="react-tabs-2"
      role="tab"
      tabIndex={null}
    >
      <span
        className="c3"
      >
        Details
      </span>
    </li>
  </ul>
  <div
    aria-labelledby="react-tabs-0"
    className="c4 is-selected"
    id="react-tabs-1"
    role="tabpanel"
  >
    <div
      style={
        Object {
          "height": "100%",
          "overflow": "hidden",
          "position": "relative",
          "width": "100%",
        }
      }
    >
      <div
        style={
          Object {
            "WebkitOverflowScrolling": "touch",
            "bottom": 0,
            "left": 0,
            "marginBottom": 0,
            "marginRight": 0,
            "overflow": "scroll",
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <span
          className="c5 c6"
        >
          Thu.oct 17
        </span>
        <div
          className="c7"
        >
          <button
            className="c8"
            height="24px"
            width="24px"
          >
            <span
              className="c5"
            >
              SP
            </span>
          </button>
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              1Sara Parker
            </span>
            <span
              className="c3 c11"
            >
              Uploaded "aportman family"
            </span>
            <span
              className="c3 c12"
            >
              10:32
            </span>
          </div>
        </div>
        <span
          className="c5 c6"
        >
          Wed.oct 16
        </span>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              2Amili Hilton
            </span>
            <span
              className="c3 c11"
            >
              Removed 3 files
            </span>
            <span
              className="c3 c12"
            >
              10:32
            </span>
          </div>
        </div>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              3Andrew Sunny
            </span>
            <span
              className="c3 c11"
            >
              Shared "aportman family" with Peter Brown.
            </span>
            <span
              className="c3 c12"
            >
              10:32
            </span>
          </div>
        </div>
        <span
          className="c5 c6"
        >
          Mon.oct 14
        </span>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              4Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              7:57
            </span>
          </div>
        </div>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              5Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              4:57
            </span>
          </div>
        </div>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              6Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              2:57
            </span>
          </div>
        </div>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              7Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              0:57
            </span>
          </div>
        </div>
        <span
          className="c5 c6"
        >
          Sun.oct 13
        </span>
        <div
          className="c7"
        >
          <button
            className="c13"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              8Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              14:07
            </span>
          </div>
        </div>
        <div
          className="c7"
        >
          <button
            className="c14"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              8Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              12:07
            </span>
          </div>
        </div>
        <span
          className="c5 c6"
        >
          Sat.oct 12
        </span>
        <div
          className="c7"
        >
          <button
            className="c14"
            height="24px"
            width="24px"
          />
          <div
            className="c9"
          >
            <span
              className="c3 c10"
            >
              9Pablo Neroda
            </span>
            <span
              className="c3 c11"
            >
              Commented on "winter fall"
            </span>
            <span
              className="c3 c12"
            >
              2:57
            </span>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "borderRadius": 3,
            "bottom": 2,
            "display": "none",
            "height": 6,
            "left": 2,
            "opacity": 0,
            "position": "absolute",
            "right": 2,
            "transition": "opacity 200ms",
          }
        }
      >
        <div
          style={
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": "inherit",
              "cursor": "pointer",
              "display": "block",
              "height": "100%",
              "position": "relative",
            }
          }
        />
      </div>
      <div
        className="c15"
        style={
          Object {
            "display": "none",
            "opacity": 0,
            "position": "absolute",
            "transition": "opacity 200ms",
            "width": 6,
          }
        }
      >
        <div
          className="c16"
          style={
            Object {
              "display": "block",
              "position": "relative",
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    aria-labelledby="react-tabs-2"
    className="c4"
    id="react-tabs-3"
    role="tabpanel"
  />
</div>
`;
